MVVMFromMVC/Controllers/WeatherViewController.swift

```diff
@@ -27,107 +27,105 @@
 /// THE SOFTWARE.
 
 import UIKit
+import RxSwift
 
 class WeatherViewController: UIViewController {
-    private let geocoder = LocationGeocoder()
-    //private let defaultAddress = "McGaheysville, VA"
+  private let geocoder = LocationGeocoder()
+  private let viewModel = WeatherViewModel()
+  
+  private let disposeBag = DisposeBag()
+  
+  @IBOutlet weak var cityLabel: UILabel!
+  @IBOutlet weak var dateLabel: UILabel!
+  @IBOutlet weak var currentIcon: UIImageView!
+  @IBOutlet weak var currentSummaryLabel: UILabel!
+  @IBOutlet weak var forecastSummary: UITextView!
+  
+  override func viewDidLoad() {
+    viewModel.locationName
+      .bind(to: cityLabel.rx.text)
+      .disposed(by: disposeBag)
+    
+    viewModel.date
+      .bind(to: dateLabel.rx.text)
+      .disposed(by: disposeBag)
+    
+    viewModel.icon.subscribe(onNext: { [weak self] icon in
```

Comment:
> なお、他のUIと同様にbindで記載したかったのですが、UIImageView型の要素に直接String型を受け渡す記載方法が分かりませんでしたので、subscribeでのデータ連携としております。 （編集済み） 

mapでUIImageに変換することで、imageにbind可能かと思います。

```suggestion
    viewModel.icon
      .map { UIImage(named: $0) }
      .bind(to: currentIcon.rx.image)
      .disposed(by: disposeBag)
```

Reply:
mapで変換することで、実装できました。